cli/catalog-api-v1/src/error.rs

```diff
@@ -14,5 +13,28 @@ pub enum MessageType {
     #[serde(rename = "constraints_too_tight")]
     ConstraintsTooTight,
 
+    #[serde(untagged)]
     Unknown(String),
 }
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    #[ignore = "useful when developing"]
```

Comment:
I can drop or run these if we prefer, but they'll be covered by re-generating mock data

Reply:
It think it would be useful to use this as a value explicitly, to signal its used as a value/constant.


```
let map: HashMap<String, MessageType> = serde_json::from_value(&json!({
 "known_type": "constraints_too_tight",
 "unknwon_type": "something unknown"
})).unwrap();

assert_eq!(map["known_type", MessageType::ConstraintsTooTight);
assert_eq!(map["unknown_type", MessageType::Unknown("something unknown".to_string()));

```

Reply:
Merged the two tests into what you suggested